csharp/ql/src/utils/model-generator/CaptureSummaryModels.qll

```diff
@@ -32,9 +15,9 @@ string captureFlow(TargetAPI api) {
 string captureQualifierFlow(TargetAPI api) {
   exists(ReturnStmt rtn |
```

Comment:
This should be using the shared data-flow library's concept of `ReturnNode`s instead. I.e., something like
```
exists(ReturnNodeExt ret |
  api = getNodeEnclosingCallable(ret) and
  isOwnInstanceAccessNode(ret)
)
```

Reply:
Alright - I will change that.

Reply:
but it does complicate things a bit as DatFlowCallable only extends Callable in C# (this is not the case for Java)